t/01_simple.t

```diff
@@ -2,16 +2,26 @@ use warnings;
 use strict;
 use Test::More tests => 22;
 use Test::TCP;
-use IO::Socket::INET;
 use t::Server;
 
+our $io_socket_module_name;
```

Comment:
Can't all of these be in t/Util.pm or whatever? Copying-and-pasting them into individual .t files should be avoided for obvious reasons.